shell/command_test.go

```diff
@@ -285,6 +285,23 @@ func TestSelectCustomShell(t *testing.T) {
 	assert.Empty(t, shell)
 }
 
+func TestRunShellWorkingDir(t *testing.T) {
+	buffer := &bytes.Buffer{}
+	cmd := NewCommand("pwd", nil)
```

Comment:
For `platform.IsWindows()` the command is `@echo %CD%` (for cmd.exe) or `(pwd).Path` for powershell. Maybe we can get away with skipping the test for the platform.

Reply:
According to the logs from GitHub Actions on the Windows platform, the command `make test-ci` was executed using PowerShell, so I opted for the `(pwd).Path` command.